specification/netapp/resource-manager/Microsoft.NetApp/preview/2017-08-15/netapp.json

```diff
@@ -1397,6 +1541,40 @@
         }
       }
     },
+    "exportPolicyRule": {
+      "description": "Volume Export Policy Rule",
+      "type": "object",
+      "properties": {
+        "ruleIndex": {
+          "type": "integer",
+          "description": "Order index"
+        },
+        "unixReadOnly": {
+          "type": "boolean",
+          "description": "Read only access"
+        },
+        "unixReadWrite": {
```

Comment:
Why is it prefixed with "unix"? Should this be included in description as well?

Reply:
It's a consequence of a lower level API that the RP calls. This value is simply passed through. But it is because it only applies to the NFS permissions. It doesn’t do anything useful for NTFS.